content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,178 @@
+:guilabel:`Description` tab.
+
+To view the new ticket, click on the link in the chat window. or go to
+:menuselection:`Helpdesk app` and click the :guilabel:`Tickets` button on the kanban card for
+the appropriate team.
+
+Search for a ticket from a live chat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If an operator types `/helpdesk_search` in the chat window, they can search through
+:guilabel:`Helpdesk` tickets by ticket number or keyword.
+
+After entering the `/helpdesk_search` command, type a keyword or ticket number, then press `Enter`.
+If a related ticket is found, a link will be generated in the conversation window.
+
+.. image:: responses/live-chat-helpdesk-search.png
+   :align: center
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
+
+History
+-------
 
-Send canned responses
-=====================
+If an operator types `/history` in the chat window, it will generate a list of the most recent
+pages the visitor has viewed on the website (up to 15).
 
-| Canned responses allow you to have a full piece of text being placed when you type a shortcut word.
-  To create them, go to :menuselection:`Live Chat --> Configuration --> Canned Responses`.
-| To use them during a conversation, simply type **:** followed by the shortcut word you created.
+.. image:: responses/live-chat-responses-history.png
+   :align: center
+   :alt: View of the results from a /history command in a Live Chat conversation.
+
+Lead
+----
+
+By typing `/lead` in the chat window, an operator can create a :guilabel:`Lead` in the
+:guilabel:`CRM` application. After typing `/lead`, create a title for this new lead, then press
+`Enter`.
 
-.. image:: responses/canned_response.png
+.. image:: responses/live-chat-responses-lead.png
    :align: center
-   :alt: View of a chat window and the use of a canned response in Odoo Live Chat
+   :alt: View of the results from a /lead command in a Live Chat conversation.
+
+The `/lead` command can only be used if the :guilabel:`CRM` app has been installed. To access the
+:guilabel:`lead`, click on the link in the chat window, or navigate to the :menuselection:`CRM app`
+to view the :guilabel:`Pipeline`.
+
+The transcript of the conversation is added to the :guilabel:`Internal Notes` tab of the
+lead form. On the :guilabel:`Extra Information` tab of the lead form, the :guilabel:`Source` will
+be listed as :guilabel:`Livechat`.
+
+Leave
+-----
+
+If an operator types `/leave` in the chat window, they can automatically exit the conversation.
 
 .. seealso::
-   - :doc:`ratings`
+   - :doc:`/applications/sales/crm/acquire_leads`
+   - :doc:`/applications/services/helpdesk/overview/getting_started`
+
+Canned responses
+================
+
+*Canned responses* are customizable inputs where a shortcut stands in for a longer response. An
+operator will enter the :guilabel:`Shortcut`, and it will automatically be replaced by the expanded
+:guilabel:`Substitution` response in the conversation.
```

Comment:
lets remove the :guilabel: from substitution, make it lowercase, and italicize it